docs/update.adoc

```diff
@@ -0,0 +1,35 @@
+= Handling new versions and updates
+:imagesdir: ./assets/images/
+:toc:
+:toc-title:
+:toclevels: 4
+
+== General
+Changing the software version of your app might require a change in agrirouter, if you decide to add new technical message types.
+
+=== When is a new certification version required?
+agrirouter is not interested in any internal changes of your software.
+However, whenever the maximum capabilities of your software change, you'll need to create a new version in the agrirouter UI and run a new certification.
```

Comment:
Same when there a capabilty which is no longer supported - my point of view.

Reply:
Checking with DKE

Reply:
Clarified by DKE; should not be changed